### ISSUE TITLE
Settings Location

### ISSUE THREAD

**Issue by Sephirox** [NekoNova/ClearVoyanceEx#2]
Seems the location is not being saved correctly, at least for me it resets every now and then.

**Comment by NekoNova** [comment]
Yes, and that's not something I can fix at the moment....
This is a race condition between the Addon loading, initializing and the settings being restored.
Had a discussion on the boards about it.

Unless I completely rewrite it, no direct fix.
Might do this for Drop 6

**Comment by Sephirox** [comment]
Interesting I mean it work perfect for version 2.0 but well I guess I'll just copy pasye and reload UI everytime.

Thanks anyways.

Sincerely,

Juan Alvarez

**Comment by NekoNova** [comment]
yeah that's what is so confusing me.
The behaviour is not consistent at all.

**Comment by Sephirox** [comment]
Yeah, weird. Well thanks anyways! I guess I'll work with it for now!

Sincerely,

Juan Alvarez
